File: sfhwebservice/pojos/comuna.php
<?php
class Comuna
{
	public $idComuna;
	public $idRegion;
	public $nombreComuna;

	function initClass($idComuna, $idRegion, $nombreComuna)
	{
		$this->idComuna = $idComuna;
		$this->idRegion= $idRegion;
		$this->nombreComuna = $nombreComuna;
	}
}
?>

File: SFH-Modulo-Escritorio-crossline/ObjectsBeans/Comuna.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ObjectsBeans
{
    public class Comuna
    {
        #region Campos
        private int idComuna;
        private int idRegion;
        private string nombreComuna;
        #endregion

        #region Constructor
        public Comuna() { 
        
        }
        #endregion

        #region Propiedad

        public int IdComuna
        {
            get { return idComuna; }
            set { idComuna = value; }
        }

        public int IdRegion
        {
            get { return idRegion; }
            set { idRegion = value; }
        }

        public string NombreComuna
        {
            get { return nombreComuna; }
            set { nombreComuna = value; }
        }
        #endregion
    }
}
